cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/сonsentStatus/ConsentStatusParamRequest.kt

```diff
@@ -0,0 +1,39 @@
+package com.sourcepoint.cmplibrary.data.network.model.optimized.сonsentStatus
+
+import com.sourcepoint.cmplibrary.data.network.util.Env
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonObject
+
+@Serializable
+internal data class ConsentStatusParamRequest(
```

Comment:
Is this class used? I believe `mobile-core` handles the call to `consent-status` already.

Reply:
Good catch!